lib/logproto/logproto-multiline-server.c

```diff
@@ -48,6 +48,9 @@ log_proto_multiline_server_new(LogTransport *transport,
                                                           options->garbage);
     case MLM_NONE:
       return log_proto_text_server_new(transport, &options->super);
+
+    default:
```

Comment:
This case should cover all constants for multiline options. This should be a g_assert_not_reached()